### DISCUSSION TITLE
List of Releases (inluding premium support releases)

### DISCUSSION THREAD

**Discussion by milesfryhover** [openssl/openssl#25146] [General Discussion]
Hi all,

I'm working on gathering a list of all OpenSSL releases for some vulnerability research that I'm doing. I've been able to find releases from multiple locations like changelog, newslog, downloads page (including the old releases), and the releases here on GH. However, I'm finding that NIST has record of releases for paid support that aren't included on any of these sources. For example, 1.0.2zg can't be found on any of the above sources but is a CPE in NIST database.

Is there potentially a more comprehensive list of all releases (including non-supported, extended support, etc) that I'm not finding?

Any help would be greatly appreciated.

**Comment by mattcaswell** [comment]
The available extended support releases are only published to our premium support customers, so you won't find them on our website. With a bit of work you might infer the release numbers from our vulnerabilities page:

https://openssl-library.org/news/vulnerabilities/index.html
